src/cljs/main/broadfcui/endpoints.cljs

```diff
@@ -447,49 +447,6 @@
   {:path (str "/groups/" group-name "/requestAccess")
    :method :post})
 
-
-(defn get-billing-projects
-  ([on-done] (get-billing-projects false on-done))
-  ([include-pending? on-done]
-   (call-ajax-orch
-    {:endpoint {:path "/profile/billing" :method :get}
-     :on-done (fn [{:keys [success? status-text get-parsed-response]}]
-                (if success?
-                  (let [pred (if include-pending?
-                               (constantly true)
-                               #(= (:creationStatus %) "Ready"))]
-                    (on-done nil (filterv pred (get-parsed-response))))
-                  (on-done status-text nil)))})))
-
-(defn get-billing-project-status [project-name on-done]
-  (get-billing-projects
-   true
-   (fn [err-text projects]
-     (if err-text
-       (on-done nil)
-       (let [project (first (filter #(= project-name (:projectName %)) projects))]
-         (on-done (:creationStatus project) (:message project)))))))
-
-(defn get-billing-accounts []
```

Comment:
While this is not technically a deprecated v1 billing project endpoint, it is no longer used after the deletion of the billing views.